dev/drivers/scripts/prep/global_det/jevs_global_det_atmos_prep.sh

```diff
@@ -38,10 +41,10 @@ export RUN=atmos
 
 export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
 export TMPDIR=$DATAROOT
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver
-export COMROOT=/lfs/h2/emc/vpppg/noscrub/$USER
+export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d/$STEP/$COMPONENT/$RUN
 
-export MODELNAME="cfs cmc cmc_regional dwd ecmwf fnmoc imd jma metfra ukmet"
+export MODELNAME="cfs cmc cmc_regional dwd fnmoc imd jma metfra ukmet ecmwf"
```

Comment:
@malloryprow I'm curious why ecmwf was moved to the back of the list of MODELNAMEs, instead of staying in alphabetical order particular reason for this change?

Reply:
We moved the job to start running at 1815Z, which is when the last ECMWF precip file comes in. I moved it to the end in case it gets in a few minutes later we can still pick it up.